model_configs/imputation/three_layer_impute.yml

```diff
@@ -6,7 +6,7 @@ lr: .001
 weight_decay: 0
```

Comment:
Do you also want to have a config in `model_configs/imputation` for the five-layer imputation network? I don't see one in that directory currently (but maybe you just forgot to track it).

Reply:
Right, I thought I had added it but apparently not